WeaveApp/resources/index.ejs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Webpack allows templating for entry point html files, so in the future we can do things like add custom variables to this for custom titles, meta tags, scripts, etc.